src/features/user/Settings/EditProfile/EditProfileForm.tsx

```diff
@@ -1,37 +1,56 @@
+import { Button, styled, TextField } from '@mui/material';
 import Snackbar from '@mui/material/Snackbar';
 import MuiAlert from '@mui/material/Alert';
 import React, { useState } from 'react';
 import { useDropzone } from 'react-dropzone';
 import { Controller, useForm } from 'react-hook-form';
-import Camera from '../../../../public/assets/images/icons/userProfileIcons/Camera';
-import CameraWhite from '../../../../public/assets/images/icons/userProfileIcons/CameraWhite';
-import { putAuthenticatedRequest } from '../../../utils/apiRequests/api';
-import COUNTRY_ADDRESS_POSTALS from '../../../utils/countryZipCode';
-import getImageUrl from '../../../utils/getImageURL';
-import { selectUserType } from '../../../utils/selectUserType';
-import AutoCompleteCountry from '../../common/InputTypes/AutoCompleteCountry';
-import MaterialTextField from '../../common/InputTypes/MaterialTextField';
-import ToggleSwitch from '../../common/InputTypes/ToggleSwitch';
-import { UserPropsContext } from '../../common/Layout/UserPropsContext';
+import Camera from '../../../../../public/assets/images/icons/userProfileIcons/Camera';
+import CameraWhite from '../../../../../public/assets/images/icons/userProfileIcons/CameraWhite';
+import { putAuthenticatedRequest } from '../../../../utils/apiRequests/api';
+import COUNTRY_ADDRESS_POSTALS from '../../../../utils/countryZipCode';
+import getImageUrl from '../../../../utils/getImageURL';
+import { selectUserType } from '../../../../utils/selectUserType';
+import AutoCompleteCountry from '../../../common/InputTypes/AutoCompleteCountryNew';
+import ToggleSwitch from '../../../common/InputTypes/ToggleSwitch';
+import { UserPropsContext } from '../../../common/Layout/UserPropsContext';
 import styles from './EditProfile.module.scss';
 import GeocoderArcGIS from 'geocoder-arcgis';
-import { ErrorHandlingContext } from '../../common/Layout/ErrorHandlingContext';
+import { ErrorHandlingContext } from '../../../common/Layout/ErrorHandlingContext';
 import { useTranslation } from 'next-i18next';
-import { handleError, APIError } from '@planet-sdk/common';
-
-interface Props {}
+import { allCountries } from '../../../../utils/constants/countries';
+import InlineFormDisplayGroup from '../../../common/Layout/Forms/InlineFormDisplayGroup';
+import {
+  MuiAutoComplete,
+  StyledAutoCompleteOption,
+} from '../../../common/InputTypes/MuiAutoComplete';
+import StyledForm from '../../../common/Layout/StyledForm';
+import { AddressSuggestionsType } from '../../../common/types/user';
+import { AlertColor } from '@mui/lab';
 
 const Alert = styled(MuiAlert)(({ theme }) => {
   return {
     backgroundColor: theme.palette.primary.main,
   };
 });
 
-export default function EditProfile({}: Props) {
+type FormData = {
+  address: string;
+  bio: string;
+  city: string;
+  firstname: string;
+  getNews: boolean;
+  isPrivate: boolean;
+  lastname: string;
+  name: string;
+  url: string;
+  zipCode: string;
+  type?: string;
```

Comment:
`type` should not be included within `FormData` as the field providing this is not registered with react hook form. In `saveProfile` we get the value for `type` from state, not from form data.